VMBackup/main/backuplogger.py

```diff
@@ -43,6 +48,8 @@ def enforce_local_flag(self, enforced_local):
 
     """description of class"""
     def log(self, msg, local=False, level='Info'):
+        if(self.enforced_local_flag_value == False and self.logging_off == True):
```

Comment:
yes but if that is the issue then the backup should always fail and not intermittently. right?

Reply:
We can't say surely. It might be that during some runs a resources is locked by some process etc., which is needed to write to console (while in other runs when it succeed, it might not be locked). Similar to memory as it is succeeding also sometimes.
I am just saying it looks to be worth trying because there are below kind of sequence also of traces in customer logs, which does not explain how memory getting freed up in between of freeze (in this run Freeze continued upto **00:35:45**)
2020/01/20 00:38:21 2020-01-20 **00:34:45.047363**  Info  freezed 
2020/01/20 00:38:21 2020-01-20 **00:35:13.795149**  Info  ****** 4. Freeze Completed (Signal=1 received) 
2020/01/20 00:38:21 2020-01-20 00:37:09.995405  Info  some child process terminated

Reply:
I am not suggesting that logging_off change should not be done. I think it will be useful anyways. But, asking to also check if console change works for this particular customer.

Reply:
ok.. will share the same private to the customer.